.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,35 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: The name of the tarantool build artifact
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone the tarantool-python connector
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.repository_owner }}/tarantool-python
+
+      - name: Download the tarantool build artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ inputs.artifact_name }}
+
```

Comment:
Shouldn't we follow the recommendation to use the `setup-python` action?

> Using the `setup-python` action is the recommended way of using Python with GitHub Actions because it ensures consistent behavior across different runners and different versions of Python.

https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#specifying-a-python-version

Reply:
On the one hand, `ubuntu-20.04` GitHub runner already has Python3 inside and it works fine. So no setup actions are needed. On the other hand, I don't see anything bad in this recommendation, moreover, I see a few prons of it:
* Python version is pinned and we explicitly see it
* if we move to self-hosted runners, we will not be worry about Python